spec/unit/resource/powershell_script_spec.rb

```diff
@@ -130,24 +130,7 @@
     let(:resource_instance_name ) { @resource.command }
     let(:resource_name) { :powershell_script }
     let(:interpreter_file_name) { "powershell.exe" }
-    before do
-      allow(@resource).to receive(:default_flags).and_return(nil)
-    end
-    it_behaves_like "a Windows script resource"
-  end
-
-  describe "#flags" do
-    let(:resource) { @resource }
 
-    it "appends user's flags to the defaults" do
```

Comment:
This was a particular bug that we wanted to make sure we didn't introduce a regression for. If someone has an option that was basically foo false and default set it to true we want to make sure we're always applying the user supplied values last. We need some sort of test to make sure refactoring later doesn't break the order.

Reply:
Good call. I can probably add a test for the command output.